lib/ts/apidefs.h.in

```diff
@@ -1125,6 +1173,20 @@ extern tsapi const char *const TS_NPN_PROTOCOL_GROUP_SPDY;
  */
 extern tsapi const TSMLoc TS_NULL_MLOC;
 
+/* --------------------------------------------------------------------------
+   Interface for the UUID APIs. https://www.ietf.org/rfc/rfc4122.txt. */
+typedef enum {
+  TS_UUID_UNDEFINED = 0,
+  TS_UUID_V1 = 1,
+  TS_UUID_V2,
+  TS_UUID_V3,
+  TS_UUID_V4, /* At this point, this is the only implemented version (or variant) */
+  TS_UUID_V5,
+} TSUuidVersion;
+
+#define TSUuidStringLen 36
```

Comment:
This should be ``TS_UUID_STRING_LEN`` to be consistent with ``TS_MAX_USER_NAME_LEN``, which is the only precedent I could find :)